src/commands/auth.ts

```diff
@@ -7,7 +7,12 @@ import validator from 'validator';
 
 const AuthenticationHandler = async (options, command) => {
   try {
+    if(Auth.getLoggedInState()) {
```

Comment:
If a user is already logged in API will send` FDK-0003 - You are already logged In`

Reply:
@vivek8690  Yes, but for an already logged in user why is there a need for password input when we are storing user credentials in Config store. We can directly check from the store itself if he is logged in state or not.